service/logger/audit/logger_test.go

```diff
@@ -84,13 +84,6 @@ var policyCRUDParams = PolicyEventParams{
 	ActionType: ActionTypeUpdate,
 	ObjectID:   "test-object-id",
 	ObjectType: ObjectTypeKeyObject,
-
```

Comment:
Removing the diff logic test here because it is actually created by code in the policy.go file, so these tests don't even need to test that logic